great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -75,7 +73,7 @@
     # Sometimes "snowflake-sqlalchemy" fails to self-register in certain environments, so we do it explicitly.
     # (see https://stackoverflow.com/questions/53284762/nosuchmoduleerror-cant-load-plugin-sqlalchemy-dialectssnowflake)
     registry.register("snowflake", "snowflake.sqlalchemy", "dialect")
-except (ImportError, KeyError):
+except (ImportError, KeyError, AttributeError):
```

Comment:
What triggers the attribute error here?

Reply:
`registry` can be None if if there is no SqlAlchemy imported. This was the error that was being shown to the users.

Reply:
Got it... thanks!